pages/resources/collectibles.mdx

```diff
@@ -0,0 +1,802 @@
+| type    | integer                                      | The [collectible product type](#collectible-product-type)    |
+| id      | snowflake                                    | The ID of the nameplate                                      |
+| sku_id  | snowflake                                    | The SKU ID of the nameplate                                  |
+| palette | string                                       | The nameplate's [color palette](#nameplate-color-palette)    |
+| asset   | string                                       | The [nameplate asset path](/reference#cdn-formatting)        |
+| assets  | [item assets](#item-assets-structure) object | The URLs for the static and animated images of the nameplate |
+| label   | string                                       | The nameplate accessibility description                      |
+
+###### Nameplate Color Palette
+
+| Value      | Name      |
+| ---------- | --------- |
+| none       | None      |
+| crimson    | Crimson   |
+| berry      | Berry     |
+| sky        | Sky       |
+| teal       | Teal      |
+| forest     | Forest    |
+| bubble_gum | BubbleGum |
+| violet     | Violet    |
+| cobalt     | Cobalt    |
+| clover     | Clover    |
+| lemon      | Lemon     |
+| white      | White     |
+
+###### Item Assets Structure
+
+| Field              | Type   | Description                                                        |
+| ------------------ | ------ | ------------------------------------------------------------------ |
+| static_image_url   | string | The URL for the static image of the collectible                    |
+| animated_image_url | string | The URL for the animated image of the collectible (in APNG format) |
+| video_url?         | string | The URL for the video of the collectible                           |
+
+### Shop Block Object
+
+###### Hero Shop Block Structure
+
+| Field                     | Type                                           | Description                                                                                     |
+| ------------------------- | ---------------------------------------------- | ----------------------------------------------------------------------------------------------- |
+| banner_asset              | [asset config](#asset-config-structure) object | The banner asset                                                                                |
+| banner_display_config     | [banner asset](#banner-asset-structure) object | The display configuration                                                                       |
+| banner_text_color?        | string                                         | The hex color code for the banner text                                                          |
+| category_sku_id           | snowflake                                      | The [collectible category](#collectible-category-structure) SKU ID                              |
```

Comment:
```suggestion
| category_sku_id           | snowflake                                      | The collectible category SKU ID                              |
```
unnecessary to link object when it isn't referenced (apply to rest of fields too)